src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -41,8 +42,10 @@ const DragDropContextWrapper: FC<dragDropProps> = ({
   const [isTaskOnDrag, setIsTaskOnDrag] = useState<boolean>(false);
 
   useEffect(() => {
-    setTaskList(unAssignedTasks);
-    setMemberList(idleMembers);
+    setTaskList(appendNewTasks(taskList, unAssignedTasks));
+    const newMembers = idleMembers.filter((member) => !memberList.includes(member));
+    const newMemberList = [...memberList, ...newMembers];
```

Comment:
Can we put this logic in a function too, just like we did tasks in `appendNewTasks`

Reply:
done.